### ISSUE TITLE
Context menu: aria menu expanded on incorrect element

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#768]
### Expected behavior

Aria-expanded state on elements used by accessibility API to use state information. People using a screenreader hear that the button has a menu which can expand. 
### Actual behavior

Aria-expanded is on an element which isn't a role which can use this attribute/state. People using a screenreader don't hear that the button has a menu which can expand. 
https://www.w3.org/TR/wai-aria-1.1/#aria-expanded

Should probably also move the aria-haspopup property to the button element that receive focus (and has role in HTML).

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.0](https://github.com/blackbaud/skyux/releases/tag/v1.7.0)
